ios/Classes/CamerawesomePlugin.m

```diff
@@ -398,4 +411,20 @@ - (void)_handleGetTextures:(FlutterMethodCall*)call result:(FlutterResult)result
     result(@(_textureId));
 }
 
+- (FlutterResult)wrappedResult:(FlutterResult)result method:(NSString *)method {
+//    # Uncomment this code to easily debug performance issues
```

Comment:
Is it possible to wrap this in an if(DEBUG) statement (or something similar)? I think it's fine to always have this enabled during development.

Reply:
Yup, I thought about adding a `setDebugEnabled` method call to the lib to easily toggle this from the app, but I had already created the PR. Will add this to the PR before merging.